generators/app/prompts.js

```diff
@@ -163,7 +163,7 @@ function askForMoreModules() {
 
 function askModulesToBeInstalled(done, generator) {
     generator.httpsGet(
-        'https://api.npms.io/v2/search?q=keywords:jhipster-module+jhipster-5&from=0&size=50',
+        'https://api.npms.io/v2/search?q=keywords:jhipster-module+jhipster-6&from=0&size=50',
```

Comment:
I think we should compute this dynamically, using the major number from package.json version

Reply:
You're right, this is fixed!